core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -405,6 +405,15 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
     runEvent(JobCancelled(jobId, None))
   }
 
+  /** Make task set success and check result. */
```

Comment:
It doesn't make the whole taskset success but only some tasks in the taskset success?

Reply:
You said right, I will change the comments.